### PR TITLE
Milestone 1: Setup project

### PR DESCRIPTION
**Pull request by mdg0410**
> In this milestone: 

- Add the basic HTML structure as per project requirements
- Add basic styling
- Add method render

> Link to my project
https://mdg0410.github.io/Leaderboard/

### CODE REVIEW

File: src/modules/display.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  scores = [];
+
+  localData = 'Score';
+
+  workForm = '';
+
+  constructor() {
+    this.container = document.getElementById('listScore');
+    if (localStorage.getItem(this.localData)) {
+      // eslint-disable-next-line max-len
```
Comment (AlphaNtihinduka) [inline src/modules/display.js]: - [ ] Please let's avoid disabling linters. Linters can identify certain programming errors earlier on that you may not have noticed while writing code ☺️.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @mdg0410     🙋‍♂️
Good job working on your leaderboard setup. You have done incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Used webpack to bundle Javascript ✔️ 
- Followed the wireframe ✔️ 
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Optional suggestions
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: src/modules/display.js (hunk 1 above)
Comment (mdg0410) [reply src/modules/display.js]: > Correction

The `eslint-disable-next-line max-len` error is resolved by splitting the line into three parts

https://github.com/mdg0410/Leaderboard/blob/393f91acdae939a7c38cc03a6ab9183289f1d60a/src/modules/display.js#L13
https://github.com/mdg0410/Leaderboard/blob/393f91acdae939a7c38cc03a6ab9183289f1d60a/src/modules/display.js#L14
https://github.com/mdg0410/Leaderboard/blob/393f91acdae939a7c38cc03a6ab9183289f1d60a/src/modules/display.js#L15

**Review by Davidosky007 — APPROVED** [review]
Hi @mdg0410 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Webpack setup properly 👍 
* ES6 modules  used to write modular JavaScript 👍 
* Good looking readme 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**